app/views/Dashboard/Country/index.tsx

```diff
@@ -17,57 +17,79 @@ import {
     TextOutput,
     ListView,
 } from '@the-deep/deep-ui';
-import StatusCard, { Props as StatusCardProps } from '#components/StatusCard';
 
 import IndicatorChart from '#components/IndicatorChart';
 import PercentageStats from '#components/PercentageStats';
-import CustomLabel from '#components/CustomLabel';
+import ReadinessCard from '#components/ReadinessCard';
 import {
     indicatorData,
     outbreakData,
     statusData,
     genderDisaggregationData,
+    readinessData,
+    PercentageStatsProps,
+    ReadinessCardProps,
 } from '#utils/dummyData';
 
 import styles from './styles.css';
 
-const keySelector = (d: StatusCardProps) => d.statusId;
+const percentageKeySelector = (d: PercentageStatsProps) => d.id;
+const readinessKeySelector = (d: ReadinessCardProps) => d.id;
 
 interface CountryProps {
     className?: string;
 }
 
-const COLORS = ['#D7DF23', '#616161', '#00ACC1'];
+const COLORS = ['#567968', '#52625A', '#AFFAD5'];
 
 function Country(props: CountryProps) {
     const {
         className,
     } = props;
 
-    const rendererParams = useCallback((_, data: StatusCardProps) => ({
-        statusId: data.statusId,
+    const rendererParams = useCallback((_, data: PercentageStatsProps) => ({
```

Comment:
Let's name this  properly
```suggestion
    const statusRendererParams = useCallback((_, data: PercentageStatsProps) => ({
```

Reply:
Done